examples/kafka/README.md

```diff
@@ -0,0 +1,60 @@
+# kafka Project
```

Comment:
Is this README generated?
The commands described in here won't work because the `./gradlew` executable is in the repository root.
If it is generated it should be replaced with something that describes the actual context of this example (see other modules for reference)